src/dataSources/cloudFirestore/session.js

```diff
@@ -88,8 +120,11 @@ function sessions(dbInstance) {
     if (slug) {
       scrubbedSession.slug = slug;
     }
-    dlog('saving session %o', scrubbedSession);
 
+    // mentions
+    await handleMentions({ scrubbedSession, firestore: dbInstance });
```

Comment:
Total nit, but handleMentions is unlcear as to what it really does.

Reply:
Something more like, `handleSessionMentions`?

Reply:
I guess I don't know what `handle` means. It's hard for me not to think of an event handler.

Reply:
I guess what it is doing is parsing tokens out of text, looking up in slug collection and adding community slugs to session.communities. With that perhaps `parseSessionMentions` work. Thoughts @csell5 ?

Reply:
parseAndPersistMentions? like I said total nit.

Reply:
Not sure what persists means here

Reply:
`parseAndSetMentions`?